package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "glimpse.client.hud",
-  "version": "0.17.0",
+  "version": "0.17.3",
```

Comment:
client is currently at 0.17.2, so I wanted all the repros to be in sync at 0.17.3.